### PR TITLE
Package refactor

### PR DESCRIPTION
**Pull request by tuvaforrest**
## Describe your changes

Please include a summary of any changes.
-- updated for package workflow 

## How has this been tested?
-- run in multiple environments with varied configurations

Please describe the tests you ran to verify your changes.  Provide instructions to reproduce or code to reproduce output.

## Reviewer focus

Please summarize the specific items you’d like the reviewer(s) to look into.

## Checklist before requesting a review

- [ ]  I have recorded a Loom performing a self-review of my code
- [ ]  My code follows style guidelines
- [ ]  I have commented my code as necessary
- [x]  My code `dbt build` without errors in Snowflake
- [ ]  My code `dbt build` without errors in Redshift
- [x]  I have implemented generic dbt tests to validate primary keys/uniqueness in my model
- [ ]  I have updated dbt docs by running `dbt docs generate` and copying the appropriate files to the `docs/` path

### CODE REVIEW

File: models/_source.yml [hunk 1]
```diff
@@ -93,8 +93,8 @@ sources:
...
-  - name: tuva_terminology      # configuration required
-    database: tuva              # name of the terminology database
-    schema: terminology         # name of the terminology schema
-    tables:
-      - name: chronic_conditions
+#  - name: tuva_terminology      # configuration required
+#    database: tuva              # name of the terminology database
+#    schema: terminology         # name of the terminology schema
+#    tables:
+#      - name: chronic_conditions
```
Comment (sarah-tuva) [inline models/_source.yml:100]: Should this be removed instead of just commented out?

File: integration_tests/models/source.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: 2
+
+sources:
+  - name: core
+    database: "{{ var('data_profiling_input_database','tuva') }}"
+    schema:  "{{ var('data_profiling_input_schema','core') }}"
```
Comment (sarah-tuva) [inline integration_tests/models/source.yml:6]: I think these vars are wrong. Shouldn't they be `chronic_conditions_input_database ` and `chronic_conditions_input_schema ` instead?

File: models/_source.yml (hunk 1 above)
Comment (tuvaforrest) [reply models/_source.yml:100]: fixing

File: integration_tests/models/source.yml (hunk 2 above)
Comment (tuvaforrest) [reply integration_tests/models/source.yml]: yup!  also prescription instead of medicaion.  its fixed meow.

**Pull request merged**